### ISSUE TITLE
Create function onPending() to handle new message requests from outside users

### ISSUE THREAD

**Issue by TheLastGimbus** [carpedm20/fbchat#508]
## Description
There seems to be no good way to keep listening for pending messages. You can fetch them with `.fetchThreadList(thread_location=ThreadLocation.PENDING)`,
but not listen to them.

Should I just check pending messages every 10 seconds or what?

**Comment by madsmtm** [comment]
Just checked, this is possible to add.

To the future implementer, this is recevied as a `/t_ms` delta, and the data looks like this:
```json
{
  "class": "ThreadFolder",
  "folder": "FOLDER_PENDING",
  "irisSeqId": "4321",
  "irisTags": [
    "DeltaThreadFolder",
    "is_from_iris_fanout"
  ],
  "requestContext": {
    "apiArgs": {}
  },
  "threadKey": {
    "otherUserFbId": "1234"
  }
}
```

**Comment by TheLastGimbus** [comment]
Okay, looks pretty easy to write down.
Sorry, I never contributed to someone else's repo. What should I do if if would want to implement this for the next version (1.9.5 if i'm right)? Make new branch form v1 branch and then do pull request to v1?

**Comment by madsmtm** [comment]
Just do exactly as you said 😁

> Make new branch form v1 branch and then do pull request to v1?

I think the CONTRIBUTING.md is outdated, but you also need to install and run [`black`](https://github.com/psf/black) on your code before committing (otherwise CI won't pass 😉)

**Comment by madsmtm** [comment]
Aight, this is merged and released in `v1.9.5`
